src/main/kotlin/org/pkl/intellij/packages/dto/PackageUri.kt

```diff
@@ -98,9 +98,17 @@ data class PackageUri(
 
   private val lastSegmentName = path.substringAfterLast('/').substringBeforeLast("::")
 
-  val relativeZipFile = "package-1/$authority$basePath@$version/$lastSegmentName.zip"
+  val relativeZipFiles =
+    listOf(
+      "package-2/$authority$basePath@$version/$lastSegmentName.zip",
+      "package-1/$authority$basePath@$version/$lastSegmentName.zip"
+    )
 
-  val relativeMetadataFile = "package-1/$authority$basePath@$version/$lastSegmentName.json"
+  val relativeMetadataFiles =
+    listOf(
+      "package-2/$authority$basePath@$version/$lastSegmentName.json",
+      "package-1/$authority$basePath@$version/$lastSegmentName.json"
+    )
```

Comment:
How much faith do we have in this repetition being JITed out?
```suggestion
  private val packageFilenameBase = "$authority$basePath@$version/$lastSegmentName"
  
  val relativeZipFiles =
     listOf(
       "package-2/$packageFilenameBase.zip",
       "package-1/$packageFilenameBase.zip"
     )

   val relativeMetadataFiles =
     listOf(
       "package-2/$packageFilenameBase.json",
       "package-1/$packageFilenameBase.json"
     )
```